filebeat/module/elasticsearch/audit/test/test.log-expected.json

```diff
@@ -1,107 +1,107 @@
 [
     {
         "@timestamp": "2018-06-19T05:16:15,549", 
-        "elasticsearch.audit.event_type": "authentication_failed", 
         "elasticsearch.audit.layer": "rest", 
-        "elasticsearch.audit.origin_address": "147.107.128.77", 
         "elasticsearch.audit.principal": "i030648", 
-        "elasticsearch.audit.uri": "/_xpack/security/_authenticate", 
         "event.dataset": "audit", 
         "event.module": "elasticsearch", 
+        "event.type": "authentication_failed", 
         "input.type": "log", 
         "log.offset": 0, 
         "message": "[2018-06-19T05:16:15,549] [rest] [authentication_failed]        origin_address=[147.107.128.77], principal=[i030648], uri=[/_xpack/security/_authenticate]", 
-        "service.name": "elasticsearch"
+        "service.name": "elasticsearch", 
```

Comment:
For later: These should be renamed to service.type.

Reply:
Out of curiosity, why is this? I've been seeing `service.name` everywhere in ECS-ready events so far.

Reply:
The definition of `service.name` and `service.type` has change slightly in ECS: https://github.com/elastic/ecs#service `service.name` could be the cluster name of Elasticsearch for example, `service.type` is the one that never changes.